nixos/modules/programs/git-worktree-switcher.nix

```diff
@@ -0,0 +1,36 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  prg = config.programs;
+  cfg = prg.git-worktree-switcher;
+
+  bashInitScript = ''
```

Comment:
This seems awfully redundant. I think we can just have one init script with the shell name substituted in

Reply:
Fixed in 3d30c8d9340903897037bdf0d8f4eed08e56fa12